apollo-runtime/src/main/java/com/apollographql/apollo/internal/RealApolloCall.java

```diff
@@ -153,6 +149,20 @@ public RealApolloCall(Operation operation, HttpUrl serverUrl, Call.Factory httpC
 
   @Override @Nonnull public RealApolloCall<T> clone() {
     return new RealApolloCall<>(operation, serverUrl, httpCallFactory, httpCache, httpCacheControl, moshi,
-        responseFieldMapper, customTypeAdapters, apolloStore, cacheControl, dispatcher, logger);
+        responseFieldMapper, customTypeAdapters, apolloStore, cacheControl, dispatcher, logger,
+        applicationInterceptors);
+  }
+
+  private List<ApolloInterceptor> getInterceptors() {
```

Comment:
just a nit: we usually omit `get` prefixes. So I guess we can make this method to return `RealApolloInterceptorChain` so to have smth like this:
```
private ApolloInterceptorChain prepareInterceptorChain(Operation operation, List< ApolloInterceptor> applicationInterceptors) {
...
return new RealApolloInterceptorChain(operation, interceptors);
}
```